db/migrations/1657547424_create_subjects_table.up.sql

```diff
@@ -2,20 +2,21 @@ BEGIN;
 
 CREATE TABLE subjects
 (
-    id              bigserial       CONSTRAINT subjects_pk PRIMARY KEY,
+    id              bigint          PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     uuid            uuid            DEFAULT uuid_generate_v4() NOT NULL,
     code            varchar         NOT NULL UNIQUE,
     name            varchar         NOT NULL,
     objective       text            NULL,
-    credit          decimal         NULL,
-    workload        decimal         NULL,
+    credit          float           NULL,
+    workload        float           NULL,
     created_at      timestamp       DEFAULT now() NOT NULL,
     updated_at      timestamp       DEFAULT now() NOT NULL,
+    published_at    timestamp       DEFAULT now() NOT NULL,
```

Comment:
(non-blocker): Not sure, but I think that `published_at` should not have a default value, I could be wrong but I think it was to identify whether or not it was in draft.